### ISSUE TITLE
Compilation Warning

### ISSUE THREAD

**Issue by maennchen** [kipcole9/cldr_units#26]
## Warning

```
==> ex_cldr_units
Compiling 20 files (.ex)
warning: Phoenix.HTML.Safe.to_iodata/1 defined in application :phoenix_html is used by the current application but the current application does not depend on :phoenix_html. To fix this, you must do one of:

  1. If :phoenix_html is part of Erlang/Elixir, you must include it under :extra_applications inside "def application" in your mix.exs

  2. If :phoenix_html is a dependency, make sure it is listed under "def deps" in your mix.exs

  3. In case you don't want to add a requirement to :phoenix_html, you may optionally skip this warning by adding [xref: [exclude: [Phoenix.HTML.Safe]]] to your "def project" in mix.exs

  lib/cldr/protocol/phoenix_html_safe.ex:5: Phoenix.HTML.Safe.Cldr.Unit.to_iodata/1
```

## Deps Versions

```console
$ mix deps | grep cldr
* cldr_utils 2.17.1 (Hex package) (mix)
  locked at 2.17.1 (cldr_utils) 052e0c2c
* ex_cldr 2.26.1 (Hex package) (mix)
  locked at 2.26.1 (ex_cldr) b666dd85
* ex_cldr_calendars 1.18.0 (Hex package) (mix)
  locked at 1.18.0 (ex_cldr_calendars) 5b47bf4e
* ex_cldr_currencies 2.13.0 (Hex package) (mix)
  locked at 2.13.0 (ex_cldr_currencies) 64731e49
* ex_cldr_dates_times 2.11.0 (Hex package) (mix)
  locked at 2.11.0 (ex_cldr_dates_times) 36b2dd6b
* ex_cldr_lists 2.10.0 (Hex package) (mix)
  locked at 2.10.0 (ex_cldr_lists) adc040cd
* ex_cldr_numbers 2.25.0 (Hex package) (mix)
  locked at 2.25.0 (ex_cldr_numbers) 0ffe6648
* ex_cldr_units 3.12.0 (Hex package) (mix)
  locked at 3.12.0 (ex_cldr_units) 2a252abb
$ elixir --version
Erlang/OTP 24 [erts-12.2] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [jit]

Elixir 1.13.1 (compiled with Erlang/OTP 22)
```

**Comment by kipcole9** [comment]
Working on a resolution, will be resolved in the next day.

**Comment by kipcole9** [comment]
Published [ex_cldr_units version 3.12.1](https://hex.pm/packages/ex_cldr_units/3.12.1) to fix the issue by adding the `xref` option to `mix.exs`.  The changelog entry is:

### Bug Fixes

* Remove compilation warning for `Phoenix.HTML.Safe` that was emitted since the `:phoenix_html` library is not a dependency. Thanks for @maennchen for the report. Fixes #26.
